### DISCUSSION TITLE
Missing `editor::SelectPrev` command?

### DISCUSSION THREAD

**Discussion by Yevgnen** [zed-industries/zed#6498] [Feature Requests]
While there's `editor::SelectNext` command and key binding, I can not find `editor::SelectPrev` command on the command palette. Is it missing?

**Comment by JosephTLyons** [comment]
Just to update this older Discussion - Zed has an `editor: select previous` command.

<img width="529" alt="SCR-20240528-tkes" src="https://github.com/zed-industries/zed/assets/19867440/8f0123b6-4c4e-4fe8-8a3f-68681c88f9e7">
